server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -262,12 +405,29 @@ public ClusterMetadataManifest(StreamInput in) throws IOException {
         this.indices = Collections.unmodifiableList(in.readList(UploadedIndexMetadata::new));
         this.previousClusterUUID = in.readString();
         this.clusterUUIDCommitted = in.readBoolean();
-        if (in.getVersion().onOrAfter(Version.V_2_12_0)) {
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
Correct target version should be added here.

Reply:
2.13 would be the correct one, right?

Reply:
yes

Reply:
for main it should be 3.0.0 and changed to 2.13.0 when backporting to 2.x